posthog/tasks/tasks.py

```diff
@@ -40,11 +41,14 @@ def redis_heartbeat() -> None:
     autoretry_for=(
         # Important: Only retry for things that might be okay on the next try
         CHQueryErrorTooManySimultaneousQueries,
+        CeleryConcurrencyLimitExceeded,
     ),
-    retry_backoff=1,
-    retry_backoff_max=2,
-    max_retries=3,
+    retry_backoff=True,
+    retry_backoff_max=10,
+    max_retries=20,
```

Comment:
why are these increased? sounds like a way of getting a thundering herd of retries when we have issues, better just to fail on the frontend.

Reply:
My idea was to increase them since we kind of expect more errors than we did before with things retrying when hitting concurrency limits. But correct, a herd of retries is also not good.
I guess we can keep it low and see.